### DISCUSSION TITLE
Why is the Powershell profile path uppercase?

### DISCUSSION THREAD

**Discussion by JustinGrote** [Azure/CloudShell#302] [Q&A]
This messes with tools like chezmoi and doesn't match the default PS7 path. Why is it uppercase and all this work to inject an uppercase profile path rather than use the default? Is there a good reason?
https://github.com/Azure/CloudShell/blame/f9bfa1e1885672243918aa23027d939c8583116e/linux/powershell/PSCloudShellStartup.ps1#L36-L37
`'.config/PowerShell/profile.ps1'` should be `'.config/powershell/profile.ps1'`
@darrentu who committed the code.

**Comment by theJasonHelmick** [comment]
Hi Justin - thank you for the PR.  We are evaluating the change and I should have more to follow up with soon.

**Reply by JustinGrote** [reply]
Thanks Jason. Relevant PR for anyone looking at this:
https://github.com/Azure/CloudShell/pull/303
